<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ServiceUpdates::CSettingParametersWidget</class>
 <widget class="QWidget" name="ServiceUpdates::CSettingParametersWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>100</width>
    <height>460</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>100</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="leftMargin">
    <number>6</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>6</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QListWidget" name="parameterList">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="verticalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
     <property name="horizontalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <property name="iconSize">
      <size>
       <width>65</width>
       <height>65</height>
      </size>
     </property>
     <property name="movement">
      <enum>QListView::Static</enum>
     </property>
     <property name="flow">
      <enum>QListView::LeftToRight</enum>
     </property>
     <property name="isWrapping" stdset="0">
      <bool>true</bool>
     </property>
     <property name="spacing">
      <number>7</number>
     </property>
     <property name="viewMode">
      <enum>QListView::IconMode</enum>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
     <property name="currentRow">
      <number>-1</number>
     </property>
     <item>
      <property name="text">
       <string>Language</string>
      </property>
      <property name="icon">
       <iconset>
        <normalon>:/Large/Icons/Configuration/Configuration_Language.png</normalon>
        <selectedon>:/Large/Icons/Configuration/Configuration_Language_act.png</selectedon>
       </iconset>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Date/Time</string>
      </property>
      <property name="icon">
       <iconset>
        <normalon>:/Large/Icons/Configuration/Configuration_DateTime.png</normalon>
        <selectedon>:/Large/Icons/Configuration/Configuration_DateTime_act.png</selectedon>
       </iconset>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Network</string>
      </property>
      <property name="icon">
       <iconset>
        <normalon>:/Large/Icons/Configuration/Configuration_Network.png</normalon>
        <selectedon>:/Large/Icons/Configuration/Configuration_Network_act.png</selectedon>
       </iconset>
      </property>
     </item>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
